auth-backends/src/main/resources/applicationContext-security-auth-oauth2Api.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
this file is needed since the `JWT` filter is a new rudder backend, and for now, they are looked-up by the existence of that file (which is a scare from another time).